### DISCUSSION TITLE
Support for armeabi-v7a

### DISCUSSION THREAD

**Discussion by Ehab-d** [matthaigh27/ChatGPT-android-app#18] [General]
The application currently only supports `arm64-v8a` architecture. A message of “The app not installed” appears when trying to install the app on a device with `armeabi-v7a` architecture. Please add `armeabi-v7a` support. Also, this would be better by making it in a separate APK, so this will not increase the size of APK.

**Comment by sunnyswag** [comment]
armeabi-v7a is the older target, for 32 bit arm cpus. Seems that google play do not support it.

**Comment by matthaigh27** [comment]
Converted to discussion as I don't know how to do that. I also don't really want to deal with something that has no google play support. It just sounds like a hassle, especially because I can't test it.

If its something I can just enable without hurting the current app or there are lots of people needing this then I'd be happy to.
